tests/console/validate_multipath.pm

```diff
@@ -121,7 +127,7 @@ sub verify_basic_multipath_configuration {
     #   * device-mapper:   Device Mapper Tools
     #   * multipath-tools: Tools to Manage Multipathed Devices with the device-mapper
     #   * kpartx:          Manages partition tables on device-mapper devices
-    verify_packages_installed(['device-mapper', 'multipath-tools', 'kpartx']);
+    verify_packages_installed($test_data->{software});
```

Comment:
I guess i could pass `$test_data->{software}->{packages}` and handle it [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10896/files#diff-e3206f5e029b64dc6d5f68cb7390e22bR310) differently but it doesnt make a lot of difference to me.

Reply:
It is fine, we could also move the comments of the package to the test data, so we will always have them in sync.